internal/planner/planner_test.go

```diff
@@ -1163,6 +1163,44 @@ func TestGetVersionConfig_ProgressiveRolloutEdgeCases(t *testing.T) {
 			expectRampPercent: 0,
 			expectSetCurrent:  true,
 		},
+		"nil rampLastModifiedAt should not cause a panic": {
+			strategy: temporaliov1alpha1.RolloutStrategy{
+				Strategy: temporaliov1alpha1.UpdateProgressive,
+				Steps: []temporaliov1alpha1.RolloutStep{
+					{
+						RampPercentage: 10,
+						PauseDuration:  metav1.Duration{Duration: 30 * time.Second},
+					},
+					{
```

Comment:
nit: I think there's no need for multiple steps here, and doing only one would make the test shorter